lib/index.ts

```diff
@@ -39,8 +53,16 @@ export const useStoryblok = (slug, apiOptions = {}, bridgeOptions = {}) => {
   return story;
 };
 
-export const useStoryblokState = (initialStory = {}, bridgeOptions = {}) => {
-  let [story, setStory] = useState(initialStory);
+export const useStoryblokState: Function = (
+  initialStory: StoryData = {} as StoryData,
+  bridgeOptions: StoryblokBridgeConfigV2 = {},
+  preview: boolean = false
+) => {
```

Comment:
I'm not sure if it's inferred, but if not can you specify the return type of the function is StoryData?